cmd/exo/root.go

```diff
@@ -31,7 +31,12 @@ func newContext() context.Context {
 	logger := logging.Default()
 	ctx = logging.ContextWithLogger(ctx, logger)
 
-	tel := telemetry.New(ctx, &cfg.Telemetry)
+	// This telemetry object will be replaced by one that includes the
+	// installation ID when the daemon starts. For one-off commands,
```

Comment:
installation id or device id? two different things?

Reply:
it should be device id. i did an auto refactor and missed the comments.